src/kbd.rs

```diff
@@ -0,0 +1,274 @@
+use x86::io::inb;
+
+// PC keyboard interface constants
+
+const KBSTATP: u16    = 0x64;    // kbd controller status port(I)
```

Comment:
I'm gonna err on the side of `rustfmt` here and say we don't need to line the assignment operators up.